packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/add-comments/controller.js

```diff
@@ -10,7 +11,10 @@ const {
 
 /** @type {import('express').RequestHandler} */
 const addCommentsGet = (req, res) => {
-	if (!confirmInterestedPartySessionAppealReference(req)) {
+	if (
```

Comment:
feels like these checks should be middleware or shared somehow, so it's handled in one place